marklogic-data-hub/src/test/java/com/marklogic/hub/master/MasterTest.java

```diff
@@ -62,7 +62,12 @@ public void testMatchEndpoint() {
         makeInputFilePathsAbsoluteInFlow("myNewFlow");
 
         runAsDataHubOperator();
-        runFlow(new FlowInputs("myNewFlow", "1", "2"));
+        RunFlowResponse flowResponse = runFlow(new FlowInputs("myNewFlow", "1", "2"));
```

Comment:
This test, and testMasterStep, are failing locally for me, and also in Jenkins - https://jenkins.marklogic.com/blue/organizations/jenkins/Datahub_CI/detail/PR-5053/1/tests . 

It's very difficult to debug these tests. The assertion messages below only restate what the assertions state. Before the mastering step is run, there are 239 docs in the "mapping" collection. Why are 7 match results expected?

I recommend trying to rework this test as part of this PR. The scope appears to be just verifying that the match endpoint works. If so, I think all we need is a test that inserts two documents (one that will match and one that won't), one flow with a mastering step, and then uses a test document to verify that one match is returned. The assertion messages should explain why the match is expected, and why the test doc doesn't match the other document. That way, when the test fails in the future, it'll be much easier to debug. Right now, it's nearly impossible to debug because a developer is faced with a ton of analysis work to figure out why 7 matches are expected, and which ones those are. Instead of trying to document all of that, I think it'd be easier and better in the long term to just replace this test. 

I recommend replacing testMasterStep for the same reason - it's failing with a message of "At least 10 merges are expected", which doesn't inspire much confidence in how accurate merging is (i.e. is it okay if there are 10 merges, or 50 merges? If so, why?).